### ISSUE TITLE
Unlock (and lock ?) commands

### ISSUE THREAD

**Issue by lyndhurst** [trustcrypto/python-onlykey#74]
I installed the cli app to get a better user experience when it comes to unlocking the key.
I was hoping to be able to associate a shortcut to invoke a terminal window and enter my pin. Maybe even set a shortcut to lock the device.
The desktop app is very sluggish, and keyboard shortcuts do not work reliably which can get annoying.
Anyway, is the cli app only meant to configure the key, or did I miss the unlock command ?
Thanks for your time.

**Comment by sonnatager** [comment]
This would be very nice, especially the abillity to lock over the cli. It would be nice, if i can lock the OnlyKey when i lock my computer because i use it to unlock my PC.

**Comment by HenriGEIST** [comment]
Yes I vote for a "lock" command then I can set my screen saver to trigger the lock command an lock the OnlyKey at same time it lock the screen.
